src/views/HomeView.vue

```diff
@@ -1,9 +1,34 @@
 <template>
-  <div></div>
+  <main>
+    <div>Platzhalter Menü</div>
+    <div>Platzhalter Icon Name</div>
```

Comment:
Info kommt aus RegUserView.vue 1. Buchstabe vom string des Vornamen

Reply:
Könnten wir eventuell nochmal als eigenes issue aufmachen. Also die Frage, wie die Icons generiert und eingebunden werden, vorstellbar als eigene component, in der eine funktion den ersten Buchstaben vom firstname rausfiltert und dann mit css als farbiger Kreis im Hintergrund gerendert wird?